clients/admin-ui/cypress/e2e/systems-plus.cy.ts

```diff
@@ -156,6 +156,81 @@ describe("System management with Plus features", () => {
     });
   });
 
+  describe("data use", () => {
+    it("should enable legal basis editing if flexible is true", () => {
+      cy.fixture("systems/system_with_flexible_legal_basis.json").then(
+        (system) => {
+          cy.intercept("GET", "/api/v1/system/*", {
+            body: system,
+          }).as("getSystemWithFlexibleDataUses");
+        }
+      );
```

Comment:
this can be simplified since you aren't modifying the `system` obj (don't commit this directly as it's not formatted correctly, but something like)

```suggestion
      cy.intercept("GET", "/api/v1/system/*, 
        { fixture: "systems/system_with_flexible_legal_basis.json" }
      ).as("getSystemWithFlexibleDataUses");
```